tools/probers/workflow/test_workflow.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
lol i prolly would have written this in go since testrbe already is and stuff like `invocation_complete=$(echo "$invocation_resp" | jq '[.invocation[] | has("bazelExitCode")] | any')` makes me a lil queezy but I'll defer to @bduffany and @iain-macdonald on this if you want to keep the bash.

Reply:
Yeah, I agree that we should generally try to write stuff like this in go, but if you'd like to file a bug to rewrite it rather than do it now, that's fine with me. Mostly depends on how urgent you think this new prober is.